### PR TITLE
replaced back-end-development.md & text-editors.md w/ code-devepment.md

### PR DESCRIPTION
**Pull request by openprivacy**
Came here to ensure tab widths were set and security coding standards were mentioned, and I did a little more.

Questions:
- How do I run travis tests in my fork before creating a PR?
- Should we rename the "Tools" heading to "Team Tools" or "Collaborative Tools"?

### CODE REVIEW

**Comment by grugnog** [comment]
Travis runs automatically - see the "All checks have passed". Not sure about the other Q.

File: docs/engineering/code-development.md [hunk 1]
```diff
@@ -1,78 +1,100 @@
...
 * [Git Commits](#git-commits)
-* [Editor Configuration](#editor-configuration)
 * [Code Examples](#code-examples)
   * [Don't use continue in short loops](#continue)
-  * [Simplify Boolean Logic](#boolean)
+  * [Simplify Boolean Logic](#simplify-boolean-logic)
+* [Still have questions?](#still-have-questions)
 
-## <a name="general-principles"></a>General Principles
+## Popular Editors at CivicActions
```
Comment (grugnog) [inline docs/engineering/code-development.md:17]: I think this page should stay named "back-end development" - the page names should correspond to practice areas (the columns in our Trello board).

File: docs/how-we-work/tools/text-editors.md [hunk 2]
```diff
@@ -1,17 +0,0 @@
-# Text editors
```
Comment (grugnog) [inline docs/how-we-work/tools/text-editors.md:1]: I am not sure why this was moved - text editors seem more like tools than practices to me

File: docs/engineering/code-development.md [hunk 3]
```diff
@@ -1,78 +1,100 @@
...
+
+* [Atom](https://atom.io/)
+* [Emacs](https://www.gnu.org/software/emacs/): Installed on OS X and Linux by default. Also available on OS X via [Homebrew](http://brew.sh/)
+* [Nano](https://www.nano-editor.org/): Installed on OS X and Linux by default. Also available on OS X via [Homebrew](http://brew.sh/)
+* [Sublime Text](http://www.sublimetext.com/)
+* [Vim](http://www.vim.org/download.php): Installed on OS X and Linux by default. Also available on OS X via [Homebrew](http://brew.sh/)
+
+### Editor Configuration for Coding
+
+* `TAB_WIDTH = 8` // the default for most editors and printers - this is the displayed width of tabs, but is irrelevant as there shouldn't be any actual tabs!
```
Comment (grugnog) [inline docs/engineering/code-development.md:29]: Do these environment variables apply to all editors? I am not sure, but would guess not. It seems like this duplicates Drupal's documentation here also?

Comment (openprivacy) [reply docs/engineering/code-development.md:29]: I don't know - probably not. I just added back-ticks to values already in the file. I figured they were a good example at the least.

File: docs/engineering/code-development.md (hunk 1 above)
Comment (openprivacy) [reply docs/engineering/code-development.md:17]: These editors and coding suggestions in the original "back-end development" file are useful for anyone coding - front and back end.

And, um, what Trello board are you referring to?
**Update:** Oh, the [Engineering Framework](https://trello.com/b/LVtWTgkS/engineering-framework)

Matching the columns in that Trello board may not work as much of the content in code-development (was: back-end development) comes from the QA column. But I will rename back as it was and re-create the text-editors page if that's what the docs team wants. Fine with me.

File: docs/how-we-work/tools/text-editors.md (hunk 2 above)
Comment (openprivacy) [reply docs/how-we-work/tools/text-editors.md:1]: I figured all the other Tools were SaaS offerings, and Editors are downloaded applications. Also, there is a strong overlap between Editors and Development Environments. But I can see it both ways.

**Pull request closed without merging**